src/main/resources/templates/db_scripts/create-holdings-status-audit-table.sql

```diff
@@ -1,4 +1,11 @@
 ALTER TABLE holdings_status_audit
-DROP COLUMN IF EXISTS id,
-ADD COLUMN IF NOT EXISTS operation VARCHAR(20) NOT NULL,
-ADD COLUMN IF NOT EXISTS updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP;
+  DROP COLUMN IF EXISTS id,
+  ADD COLUMN IF NOT EXISTS credentials_id UUID NOT NULL,
+  ADD COLUMN IF NOT EXISTS operation VARCHAR(20) NOT NULL,
+  ADD COLUMN IF NOT EXISTS updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP;
+
+ALTER TABLE holdings_status_audit
+  DROP CONSTRAINT IF EXISTS fk_holdings_status_audit_kb_credentials,
+  ADD CONSTRAINT fk_holdings_status_audit_kb_credentials FOREIGN KEY (credentials_id) REFERENCES kb_credentials (id);
```

Comment:
```suggestion
  ADD CONSTRAINT fk_holdings_status_audit_kb_credentials FOREIGN KEY (credentials_id) REFERENCES kb_credentials (id) ON DELETE CASCADE;
```

Reply:
.. and for the rest of holdings loading tables

Reply:
fixed, thanks